task-list/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<form class= "wrapper-front" method="post">
+  <div class="button-farm">
+    <a href="/add_task"><h1>Add a new task!</h1></a>
+    <input type="submit" id="submit" name="submit" value="Complete"> |
+    <input type="submit" id="submit" name="submit" value="Modify"> |
+    <input type="submit" id="submit" name="submit" onclick="return confirm('Are you sure?')" value="Delete"> |
+    <input type="submit" id="submit" name="submit" onclick="return confirm('Are you sure?')" value="Clear Completed">
+  </div>
+  <div>
+    <% unless @sorted_tasks[:todo].nil? %>
+      <% @sorted_tasks[:todo].each do |task| %>
+        <div class="task">
+          <label><input type="checkbox" name="checked[]" value="<%= task.id %>"><span><%= link_to task.name, "/tasks/" + task.id.to_s %></span></label>
+          <div class="task-detail">
+            <%= task.description %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div>
+    <% unless @sorted_tasks[:completed].nil? %>
+      <% @sorted_tasks[:completed].each do |task| %>
+        <div class="task">
+          <label><input type="checkbox" name="checked[]" value="<%= task.id %>"><span class="complete"><%= link_to task.name, "/tasks/" + task.id.to_s %></span></label>
```

Comment:
Later on you may want to and be able to create a partial view to handle these two since they are so similar